packages/react/src/components/form/authenticating/messages.ts

```diff
@@ -4,8 +4,11 @@ import { TextConfigKey } from "../../text/constants";
 // TODO add case for password
 export function getAuthenticatingMessage(
   factor: Factor,
-  isSubmitting = false
+  formState: "initial" | "input" | "submitting" | "retry" = "initial"
 ): { title: TextConfigKey; message: TextConfigKey } {
+  const isSubmitting = formState === "submitting";
```

Comment:
Do these variables add value? Looking at the form state seems quite explicit, as the whole intent was to enable easy conditional rendering.